mz_crypt_openssl.c

```diff
@@ -270,7 +270,7 @@ void mz_crypt_sha_delete(void **handle) {
 
 typedef struct mz_crypt_aes_s {
     int32_t    mode;
-    int32_t    error;
+    unsigned long    error;
```

Comment:
Can you fix the white space alignment of the other variables in the struct to match up?